tasks/freeradius.yml

```diff
@@ -0,0 +1,156 @@
+---
+- name: Freeradius system packages
+  apt:
+    name:
+      - freeradius
+      - freeradius-rest
+    state: latest
+  notify: restart freeradius
+
+- name: Freeradius postgres packages
+  when: freeradius_sql.dialect == "postgresql"
+  apt:
+    name: freeradius-postgresql
+    state: latest
+  notify: restart freeradius
+
+- name: Freeradius mysql packages
+  when: freeradius_sql.dialect == "mysql"
+  apt:
+    name: freeradius-mysql
+    state: latest
+  notify: restart freeradius
+
+- name: SQL configuration
+  template:
+    src: freeradius/sql.j2
+    dest: "{{ freeradius_mods_available_dir }}/sql"
+    mode: 0640
+    owner: freerad
+    group: freerad
+  notify: restart freeradius
+
+- name: Enable SQL module
+  file:
+    src: "{{ freeradius_mods_available_dir }}/sql"
+    # the "1_" prefix is put on purpose to force loading the sql module
+    # before the sqlcounter module to avoid an annoying startup error
+    dest: "{{ freeradius_mods_enabled_dir }}/1_sql"
```

Comment:
@atb00ker ensure you have this file in your filesystem, the sql module must loaded before sqlcounter otherwise that error you mentioned appears. I thought I solved it with this change (I tested it).

Reply:
Hmm, this solution didn't work for me and CI is failing too, I'll check it out, it should have worked though, I did something similar (renamed `sqlcounter` to `sql-counter`) in an earlier commit which worked.

Reply:
ah ok, I did not understand that, please always leave comments in the code when you do these kind of things, if it worked we can restore it